docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,149 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time in event listener. ({issue}`27202`)
+* Add lineage support for output columns in all SELECT queries ({issue}`26241`)
+* Require JDK 25 to build and run Trino ({issue}`27153`)
+* Require the JDK Vector API to be enabled at runtime ({issue}`27340`)
+* Add support for `SET DEFAULT` and `DROP DEFAULT` statements. ({issue}`26162`)
+* Add `array_first` and `array_last` functions. ({issue}`27295`)
+* Enhance after `RemoveTrivialFilters` call `RemoveEmptyUnionBranches` to prune empty union branches. ({issue}`21506`)
+* Improve performance of {func}`array_sort` function. ({issue}`27272`)
+* Improve performance of `repeat` function. ({issue}`27369`)
+* Improve performance of data exchanges by using SIMD instructions on x86 CPUs that support the required extensions. This can be disabled by setting `experimental.blockserde-vectorized-null-suppression-strategy=NONE`({issue}`26919`)
+* Fix planning failure of certain queries where part of the plan is optimized to empty values. ({issue}`21506`)
+* Fix configuring partitioned layout for spooling protocol ({issue}`27247`)
```

Comment:
Fix server startup failure when partitioned spooling is configured.
This is spooling-filesystem plugin change. Still in General section?